source/hassio/installation.markdown

```diff
@@ -11,8 +11,8 @@ The following will take you through the steps required to install Home Assistant
 
      - [Raspberry Pi 3 Model B and B+ 32-bit][pi3-32] (recommended)
      - [Raspberry Pi 3 Model B and B+ 64-bit][pi3-64]
-     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 32-bit][pi4-32] (recommended)
-     - [Raspberry Pi 4 Model B (1 GB, 2 GB and 4 GB model) 64-bit][pi4-64]
+     - [Raspberry Pi 4 Model B (1GB, 2GB and 4GB model) 32-bit][pi4-32] (recommended)
+     - [Raspberry Pi 4 Model B (1GB, 2GB and 4GB model) 64-bit][pi4-64]
```

Comment:
Please revert the change of removing the space.

There should be a space between the value and its unit of measurement.

For reasoning and more information, see both The Microsoft & Google Style Guides on writing documentation.

- <https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/units-of-measure-terms>
- <https://developers.google.com/style/units-of-measure>

Reply:
Sounds good. Thank you for letting me know about the Google/Microsoft guidelines regarding units of measure. 

If that is the case, please be sure to correct (by adding) a space to the bottom portion regarding VMware Workstation. There is a "2GB RAM" down there, which is what prompted me to recommend standardizing the spaces to begin with.

Reply:
Feel free to correct that as you process the review comments on your PR.